src/presenter/cerebral/actions/goBackAction.test.js

```diff
@@ -0,0 +1,25 @@
+import { goBackAction } from './goBackAction';
+import { presenter } from '../../../presenter/presenter';
+import { runAction } from 'cerebral/test';
+
+describe('goBackAction', () => {
+  it('should call `back` on history', () => {
+    const mockWindow = {
+      history: () => {
+        return {
+          back: () => {
```

Comment:
I know I'm reviewing after a merge, but ... I would like to suggest 
`const backMock = jest.fn();` 
and then 
`back: backMock,`.
Then you can discard the `expect.assertions`, call `runAction()` and then 
`expect(backMock).toHaveBeenCalled()`

Reply:
That's awesome, I'll try that! Thank you.